.github/workflows/release.yml

```diff
@@ -56,13 +56,24 @@ jobs:
       - name: Building packages
         run: yarn build
 
-      - name: Create Release Pull Request or Publish to npm
+      - name: Storing release version for changeset
+        id: release_version
+        run: echo "::set-output name=value::$(./scripts/print_release_version.sh)"
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+
+      - name: Creating release pull request or publishing release to npm registry
         id: changesets
-        uses: changesets/action@v1.3.0
+        # uses: changesets/action@v1.3.0
+        uses: dotansimha/changesets-action@1bd10e16a393db098c8f537c719621962c0d5571
```

Comment:
Using this version, as it hasn't been officially released yet. This is a way also to help testing this feature and give feedback.